docs/en/assets/commonMarkdown/replicanum.md

```diff
@@ -0,0 +1,5 @@
+:::note
+
+If a StarRocks cluster in a staging environment contains only one BE, the number of replicas can be set to `1` in the `PROPERTIES` clause, such as `PROPERTIES( "replication_num" = "1" )`. The default number of replicas is 3, which is also the number recommended for production StarRocks clusters. If you want to use the default number, you do not need to configure the `PROPERTIES`.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/b72a54bc-97cb-4d4c-871b-c1227ef32ffc)
`PROPERTIES` could contain a lot other optional parameters. Here it should mean that "you do not need to configure the `replication_num` parameter".